src/Pages/NavBar.js

```diff
@@ -0,0 +1,35 @@
+import Header from './Header';
+
+const NavBar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Rockets',
+    },
+    {
+      id: 2,
+      path: '/Missions',
+      text: 'Missions',
+    },
+    {
+      id: 3,
+      path: '/My Profile',
+      text: 'My Profile',
+    },
+  ];
+  return (
+    <nav className="navBar">
+      <Header />
+      <ul className="navLinks">
+        {links.map((link) => (
+          <li key={link.id} className="navLink">
+            {link.text}
+          </li>
+        ))}
+      </ul>
+    </nav>
+  );
+};
```

Comment:
- [ ] Please set here the links so the users can switch between the pages. I would suggest you use `<navlink>` inside `<li>` to set up the links
 Example: 
`<ul >
          {
          links.map((link) => (
            <li key={link.id}>
              <NavLink to={link.path}>{link.text}</NavLink>
            </li>
          ))
        }
        </ul>`